onebusaway-android/src/main/java/org/onebusaway/android/ui/TripResultsFragment.java

```diff
@@ -333,10 +337,20 @@ void updateInfo() {
 
             mDirectionsListView.setGroupIndicator(null);
 
-            if(Application.getPrefs()
-                    .getBoolean(getString(R.string.preference_key_trip_plan_notifications), true)) {
+            Context context = Application.get().getApplicationContext();
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+                NotificationManager manager = (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
+                NotificationChannel channel = manager.getNotificationChannel(Application.CHANNEL_REALTIME_ID);
+                if(channel.getImportance() != NotificationManager.IMPORTANCE_NONE){
+                    RealtimeService.start(getActivity(), getArguments());
+                }
+            }
```

Comment:
Could you please put the else brackets on same line and add a space?

`} else {`